node/src/components/consensus/highway_core/state.rs

```diff
@@ -528,7 +533,7 @@ impl<C: Context> State<C> {
                 // greater of the two exponents, a round boundary must be between the votes.
                 let max_re = prev_vote.round_exp.max(wvote.round_exp);
                 if prev_vote.timestamp >> max_re == timestamp >> max_re {
-                    return Err(VoteError::RoundLength);
+                    return Err(VoteError::RoundExpLengthLessThanMinimum);
```

Comment:
I agree with @goral09's comment below: This error is that someone changed their round exponent in the middle of a round. Maybe `VoteError::RoundLengthChangedWithinRound`?

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/13a22ef3964dd9ad4f38e73682e5d6f46e56bd12